AI Guidelines/iOS/Scripts/generate_header.sh

```
@@ -0,0 +1,17 @@
+#!/bin/zsh
```

Shellcheck (0.10.0)

[error] ShellCheck only supports sh/bash/dash/ksh/'busybox sh' scripts. Sorry!

(SC1071)